components/02-elements/breadcrumbs/breadcrumbs.hbs

```diff
@@ -1,21 +1,29 @@
-<div class="breadcrumbs {{ class }}" {{{ attributes }}}>
-    {{#each categories}}
-        {{#if @last}}
-            <span class="breadcrumbs__item"
-                  aria-label="{{ ariaLabel }}"
-                  tabindex="0"
-            >
-                {{ name }}
-            </span>
-        {{else}}
-            <span class="breadcrumbs__item">
-                <a href="{{ href }}"
-                   class="{{ linkClass }}"
-                   aria-label="{{ ariaLabel }}"
+<nav
+    class="breadcrumbs {{ class }}" {{{ attributes }}}
+    aria-label="{{ ariaLabel }}"
+    role="navigation"
+>
+    <ol
+        class="breadcrumbs__list {{ listClass }}" {{{ listAttributes }}}
+    >
+        {{#each categories}}
+            {{#if @last}}
+                <li
+                    class="breadcrumbs__item"
```

Comment:
please add attribute `aria-current="page"` it works on li too. At least in some AT. so let's use it :)

Reply:
done

Reply:
ariaLabel is added breadcrumbs.config.js for items but not used in template which is better to use it as title or aria-label ?

Reply:
I think ariaLabel that is inside categories array for each item should be removed from config as it wasn't removed when removing form template (as we have text inside link so no need for using aria label). Title in my opinion should be used conditionally if there is a need for that (if we have link like `<a href="...">Some long te......</a>` then we can use `<a href=".." "title="Some long text full text">..... ` it's used mainly for SEO..

> The title attribute value is used to provide advisory information. ... The advisory information presented should not be identical to or very similar to the element text or alternative text. it should be use for links with non-textual content, if you use it with labeled link, please set this text advisory, not repeat main label.

Reply:
removed